src/reanimated2/layoutReanimation/web/componentUtils.ts

```diff
@@ -201,7 +226,16 @@ export function handleLayoutTransition(
     existingTransform
   );
 
-  setElementAnimation(element, animationConfig);
+  const tmpTransform = existingTransform
```

Comment:
```suggestion
  const transformCopy = existingTransform
```
sounds better for me 😅

Reply:
Done in [68f3259](https://github.com/software-mansion/react-native-reanimated/pull/5298/commits/68f325953fa2d9b6c7cffbb6b421b8387f6152a9)